.github/workflows/test-build.yml

```diff
@@ -74,6 +74,8 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          clean: false
```

Comment:
Why is this needed?

Reply:
actually, its not needed, i will remove those lines.